server/src/main/java/org/elasticsearch/action/ActionModule.java

```diff
@@ -787,6 +790,7 @@ public <Request extends ActionRequest, Response extends ActionResponse> void reg
             actions.register(PutSynonymsAction.INSTANCE, TransportPutSynonymsAction.class);
             actions.register(GetSynonymsAction.INSTANCE, TransportGetSynonymsAction.class);
             actions.register(DeleteSynonymsAction.INSTANCE, TransportDeleteSynonymsAction.class);
+            actions.register(ListSynonymsAction.INSTANCE, TransportListSynonymsAction.class);
```

Comment:
I think a better name would be `GetSynonymsSetAction` as we are not listing synonyms but rather synonymsSets

Reply:
That's a good suggestion and makes sense with the names we've been using. Thanks!

Reply:
I will leave this up to you, Carlos (may be in a follow up if you want), but the current name `GetSynonymsAction ` is also fine with me.